fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -291,7 +291,7 @@ private RdfStream getResourceTriples(final int limit, final FedoraResource resou
             //streams.add(getTriples(resource, MINIMAL));
             // Mementos already have the server managed properties in the PROPERTIES category
             // since mementos are immutable and these triples are no longer managed
```

Comment:
Note can probably go if the check for Mementos is.

Reply:
Done